prime-router/settings/organizations.yml

```diff
@@ -765,7 +765,25 @@
           - orEquals(patient_state, MD, ordering_facility_state, MD)
         translation:
           type: HL7
-          schemaName: covid-19
+          useBatchHeaders: true
+        timing:
+          operation: MERGE
+          numberPerDay: 1440
+          initialTime: 00:00
+          timeZone: EASTERN
+
+  - name: md-phd
```

Comment:
We don't need both md-doh and md-dph.
I called it md-doh in Staging, which seems correcter:   the full name is "Maryland Department of Health and Mental Hygiene".